internal/utils/tdl.go

```diff
@@ -179,6 +179,10 @@ func ConvertTwitchLiveChatToTDLChat(path string, channelName string, videoID str
 			Emoticon: nil,
 		})
 
+		// set default offset value for this live comment
+		
+		var message_is_offset = false
```

Comment:
```suggestion
		message_is_offset := false
```

The var isn't needed here, you can just define it as a false boolean.